x-pack/plugins/lists/server/services/exception_lists/utils/import/sort_import_responses.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { has } from 'lodash/fp';
+import { BulkErrorSchema } from '@kbn/securitysolution-io-ts-list-types';
+
+import { ImportResponse } from '../../import_exception_list_and_items';
+
+import { isImportRegular } from './is_import_regular';
+
+/**
+ * Helper to sort responses into success and error and report on
+ * final results
+ * @param responses {array}
+ * @returns {object} totals of successes and errors
+ */
+export const sortImportResponses = (
+  responses: ImportResponse[]
+): {
```

Comment:
I'd add return type here.

Reply:
It's there! 😉  I think the lists plugin has that linter rule to enforce it.